javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,84 @@
+let name = "";
+let toDoList = [];
+
+function getReply(command){
+    // lowercase command and split it into commandWords array
+   const commandWords = command.toLowerCase().split(" ");
+   let response = null;
+   let toDoText = "";
+
+   if (name !== "" && commandWords[0] == "hello"){
+    console.log("you've already introduced yourself") 
+    return 
+   }
+
+   switch(commandWords[0]) {
+      case "hello":
+        name = commandWords.pop();
+        response = "nice to meet you " + name; break;
+
+      case "what":
+        switch(true) {
```

Comment:
This would benefit from a longer if/else-if/else statement. 
I've never seen this use of switch-cases and I think it'd confuse colleagues to read it.

Reply:
oh ok. next time I will use them instead.

Reply:
Is it ok to merge it now and on later coding apply this method?

Reply:
If you're happy with your code you can merge :) You don't need my permission, just a review